lib/taskana-core/src/main/java/pro/taskana/task/api/TaskService.java

```diff
@@ -366,6 +376,80 @@ void forceDeleteTask(String taskId)
   List<String> updateTasks(List<String> taskIds, Map<String, String> customFieldsToUpdate)
       throws InvalidArgumentException;
 
+  /**
+   * Create a task comment.
+   *
+   * @param taskComment the task comment to be created.
+   * @return the created task comment.
+   * @throws NotAuthorizedException If the current user has no authorization to create a task
+   *     comment for the given taskId in the TaskComment.
```

Comment:
please add "or if the user is not authorized to access the task".

Reply:
done